webapp/src/selectors.ts

```diff
@@ -567,16 +569,17 @@ export const shouldPlayJoinUserSound = (state: GlobalState): boolean =>
 
 export const isOnPremNotEnterprise = (state: GlobalState): boolean => {
     const license = getLicense(state);
-    const enterprise = license.SkuShortName === LicenseSkus.E20 || license.SkuShortName === LicenseSkus.Enterprise;
+    const enterprise = license.SkuShortName === LicenseSkus.Enterprise || license.SkuShortName === LicenseSkus.EntepriseAdvanced;
```

Comment:
When did we completely remove the `E10` and `E20` sku names? (I just want to make sure we're not removing support for our minimum supported MM version.)

Reply:
E10 and E20 are legacy SKUs that were phased out several years ago, and all of our customers have since been updated to the latest SKUs. Their deprecation is being officially announced at the code level in version 10.8.here: https://github.com/mattermost/docs/pull/7888/files#diff-1c3ff347e1594f6e28919888f66ac98d74b7c69a6f45906de559bd4619e3a226

Removing it from the codebase for simplicity as is no longer needed.